submitit/local/local.py

```diff
@@ -107,7 +107,7 @@ class LocalJobEnvironment(job_environment.JobEnvironment):
     }
 
     def activated(self) -> bool:
-        return "SUBMITIT_LOCAL_JOB_ID" in os.environ
+        return os.environ.get("SUBMITIT_EXECUTOR", "") == "local"
```

Comment:
could we replace this by a default implementation in the JobEnvironment ? 

```
return os.environ.get("SUBMITIT_EXECUTOR", "") == self.name()
```

Reply:
Done